src/vivarium_census_prl_synth_pop/constants/data_keys.py

```diff
@@ -28,6 +28,7 @@ class __SyntheticData(NamedTuple):
     FIRST_NAMES: str = "synthetic_data.first_names"
     LAST_NAMES: str = "synthetic_data.last_names"
     ADDRESSES: str = "synthetic_data.addresses"
+    BUSINESS_NAMES: str = "synthetic_data.business_names"
```

Comment:
This will be used later when we want to add the names we generate to the artifact.